freqtrade/main.py

```diff
@@ -129,9 +129,16 @@ def check_handle_timedout(timeoutvalue: int) -> None:
     timeoutthreashold = arrow.utcnow().shift(minutes=-timeoutvalue).datetime
 
     for trade in Trade.query.filter(Trade.open_order_id.isnot(None)).all():
-        order = exchange.get_order(trade.open_order_id)
+        try:
+            order = exchange.get_order(trade.open_order_id)
+        except OperationalException:
```

Comment:
An OperationalException must not be ignored without giving feedback, thats its only right to exist. See docstring: https://github.com/gcarq/freqtrade/blob/70f2aed0a73fb1583b0f869fb7b0432ef9c9f068/freqtrade/__init__.py#L12 Catching `requests.exceptions.RequestException` should be enough in this case.

Reply:
Fixed!